lesson7-questions.txt

```diff
@@ -14,13 +14,13 @@ This creates several files, including two you need to know about.  What are they
 After you do this statement but before starting the rails server, you have to do another
 command.  What is that command?
 
-
+models/subscription.rb and a migration file. the necessary command is bin/rails db:migrate
 
 3. What table is created as a result of the migration described above? What are the column
 names in that table? What is the primary key? What are the foreign keys?  What does a 
 foreign key do?
 
-
+subscriptions table with columns forum, user, and priority. forum and user are foreign keys that refer to the forums and users table. the primary key is the automatically generated id column.
```

Comment:
the columns are specifically `forum_id`, etc, to point to the correct row of the other tables.